prow/plugins/trigger/generic-comment.go

```diff
@@ -65,11 +71,6 @@ func handleGenericComment(c Client, trigger plugins.Trigger, gc github.GenericCo
 		return nil
 	}
 
-	pr, err := c.GitHubClient.GetPullRequest(org, repo, number)
```

Comment:
I moved this down, saving us one github token in case an untrusted user created a `test` comment